parse/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod lockfiles;
```

Comment:
Why did you go for this crate structure? It seems rather strange to me to have everything inside a single module with that being the only thing directly exposed in the crate root.

Reply:
The parser crate was expected to have other parsers besides lock files.

Reply:
I'm not sure it's a good idea to have a "jack of all trades" parsing crate. I personally prefer specialized crates that focus on doing one thing well.

Reply:
My original crate was phylum-lockfile. I can bring that back.

Reply:
Yeah I'd prefer `phylum_lockfile` myself over `parse`.